javascript/chronometer.js

```diff
@@ -1,34 +1,41 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
-
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+      if (callback != null) callback();
```

Comment:
boa! outra opção para a condição do if é if (callback) - tudo que está dentro do parenteses do if, o interpretador vai transformar em booleano. quando a start for chamada sem nenhum parâmetro, callback será undefined -- e undefined é convertido para false. Quando ela existe, é convertido para true.